Automattic/jetpack
# Migrate My Jetpack's licenses query and state to react-query

Related https://github.com/Automattic/jetpack/pull/35697

## Proposed changes:
- Migrate licenses query and state to react-query

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

* Activate the feature branch in the Jetpack Beta plugin using Jurassic Ninja
* If you don't have any unactivated licenses, purchase some product (e.g. VideoPress) on https://cloud.jetpack.com/pricing
* Go to `/wp-admin/admin.php?page=my-jetpack#/add-license`
* You should be able to see your available license(s) in the selector input
![CleanShot 2024-02-28 at 17 27 45@2x](https://github.com/Automattic/jetpack/assets/8419292/25f1bcd8-a46c-40a9-a242-879b2bdd4dd8)
